README.md

```diff
@@ -26,18 +26,18 @@ Alternatively, you can connect your project manually as per the chapter below.
 #### Connecting to your project manually
 
 If you want to change the source Kentico Cloud project, follow these steps:
- 
-1. In Kentico Cloud, choose Project settings from the app menu.
-2. Under Development, choose API keys.
-3. Copy your Project ID
-4. Open the [`src\Client.js`](/src/Client.js) file in the sample application folder.
-5. Find the `projectId` constant.
-6. Change the values of the constant using the Project ID key you copied.
-7. Save the file.
+
+1.  In Kentico Cloud, choose Project settings from the app menu.
```

Comment:
Could you also change the lists to start with just one space to fulfill the default indentation there?

https://github.com/DavidAnson/markdownlint/blob/v0.15.0/doc/Rules.md#md030